### PR TITLE
Remove oredict for Bone Block recipe, stop white dye to bonemeal exploit

### PR DESCRIPTION
**Pull request by mezz**
Fixes issue #3176 
Tested with Botania's white dye (Floral White Powder)

### CODE REVIEW

**Comment by Actuarius** [comment]
@mezz added labels [Bug]

File: src/main/java/net/minecraftforge/oredict/OreDictionary.java [hunk 1]
```diff
@@ -336,6 +336,7 @@ private static void initVanillaEntries()
             new ItemStack(Blocks.DARK_OAK_STAIRS),
             new ItemStack(Blocks.WOODEN_SLAB),
             new ItemStack(Blocks.GLASS_PANE),
+            new ItemStack(Blocks.field_189880_di), // Bone Block, to prevent conversion of dyes into bone meal.
```
Comment (heldplayer) [inline src/main/java/net/minecraftforge/oredict/OreDictionary.java:339]: How come this uses the SRG name, compared to the rest that uses MCP names? Unless old mappings are being used, this field has a name.

Comment (kashike) [reply src/main/java/net/minecraftforge/oredict/OreDictionary.java:339]: Old mappings are being used (`20160518`).

Comment (heldplayer) [reply src/main/java/net/minecraftforge/oredict/OreDictionary.java:339]: Ah, very well then! :)

**Pull request merged**